### PR TITLE
Configurable type mapping for real estate

### LINKED ISSUE #343: Missing mapping for RealEstate.type
The federal specification defines following output for RealEstate.type:

- 'RealEstate' => e.g. Liegenschaft in BL
- 'Distinct_and_permanent_rights.BuildingRight' => e.g. 'Baurecht' in BL
- 'Distinct_and_permanent_rights.right_to_spring_water' => e.g. 'Quellenrecht' in BL
- 'Distinct_and_permanent_rights.concession' => e.g. 'Konzessionsrecht' in BL
- 'Distinct_and_permanent_rights.other' => e.g. 'weitere' in BL
- 'Mineral_rights' => e.g. 'Bergwerk' in BL

### PR DESCRIPTION
**Pull request by vvmruder**
fixes #343

### CODE REVIEW

File: pyramid_oereb/standard/pyramid_oereb.yml.mako [hunk 1]
```diff
@@ -216,6 +216,19 @@ pyramid_oereb:
         stroke_opacity: 0.6
         stroke_color: '#e60000'
         stroke_width: 5
+    type_mapping:
```
Comment (voisardf) [inline pyramid_oereb/standard/pyramid_oereb.yml.mako]: Am I right that this change means we have to adapt the configuration with our french translations (or do the mapping in fme beforehand)?

Comment (vvmruder) [reply pyramid_oereb/standard/pyramid_oereb.yml.mako]: Yes exactly. You need to map the type inside your data to the federal defined types. The system allows multi mapping.

An Example. In your data both "Grundstück" and "Liegenschaft" means "RealEstate" than you can configure it like this:
```
type_mapping:
  - mapping: RealEstate
    type: Liegenschaft
  - mapping: RealEstate
    type: Grundstück
```

This way everyone can leave its AV data like it is and configure correct mapping instead :grimacing:

Comment (vvmruder) [reply pyramid_oereb/standard/pyramid_oereb.yml.mako]: And you don't *must* configure this. If it is not configured it uses the value from data.

**Comment by vvmruder** [comment]
@jwkaltz Is this mergable from your point of view?

**Comment by jwkaltz** [comment]
Yes, thanks

**Pull request merged**